github-runner-provisioner/macm1.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+type runnerConfig struct {
+	imageId              string
+	hostResourceGroupArn string
+	placement            types.Placement
+	instanceCount        int32
+	shutdownBehavior     types.ShutdownBehavior
+	instanceType         types.InstanceType
+}
+
+var macM1HostResourceGroupArn = "arn:aws:resource-groups:us-east-1:914373874199:group/GitHub-Runners"
+
+var macM1Config = runnerConfig{
+	imageId: "ami-01b8fcd5770ceb9c1",
```

Comment:
We should add a comment here about what this AMI is (macOS 12.6 on arm64)

Reply:
done